lib/pacemaker/libpacemaker_private.h

```diff
@@ -189,6 +192,10 @@ void pcmk__order_stops_before_shutdown(pe_node_t *node,
 G_GNUC_INTERNAL
 void pcmk__apply_orderings(pe_working_set_t *data_set);
 
+G_GNUC_INTERNAL
+void pcmk__order_after_all(pe_action_t *after, GList *list);
```

Comment:
`pcmk__order_after_each` might be a better name for this function.  _after_all makes it sound like the action gets appended to the list.

Reply:
Good point, added commit on top